mvcc/watcher.go

```diff
@@ -22,8 +22,14 @@ import (
 	"github.com/coreos/etcd/mvcc/mvccpb"
 )
 
+// DefaultWatchID is the watcher ID passed in WatchStream.Watch when no
+// user-provided ID is available. If pass, an ID will automatically be assigned.
+const DefaultWatchID WatchID = 0
```

Comment:
AutoWatchID? it is more descriptive. when auto watch id is set, etcd automatically assign watch id.